src/x509.c

```diff
@@ -6711,6 +6563,14 @@ static int X509PrintValidity(WOLFSSL_BIO* bio, WOLFSSL_ASN1_TIME * notBefore,
 }
 #endif /* ifndef NO_ASN_TIME */
 
+static int X509PrintSubjAltName(WOLFSSL_BIO* bio, WOLFSSL_X509* x509,
+                                int indent)
+{
+    if (!x509 || !x509->altNames || !x509->subjAltNameSet)
+        return WOLFSSL_FAILURE;
+    return X509_ACERT_print_name_entry(bio, x509->altNames, indent);
```

Comment:
Can you rename the function since its not ACERT specific anymore?

Reply:
done